extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -768,11 +770,11 @@ private EmbeddedInfo getEmbeddedInfo() throws ParsingException, ReCaptchaExcepti
 
         } catch (IOException e) {
             throw new ParsingException(
-                    "Could load decryption code form restricted video for the Youtube service.", e);
+                    "Could load deobfuscation code form restricted video for the Youtube service.", e);
```

Comment:
Mmmh, this should probably be "Could not load deobfuscation code for restricted video"

Reply:
Sounds good. Feel free to open a pn

Reply:
#444